Airplane-tickets-UA.Shared/FlightHelper.cs

```diff
@@ -71,10 +67,9 @@ private List<ExistingFlight> GetSpecialOffer(List<ExistingFlight> existingFlight
                         existingFlight.Price *= _discound1;
                         break;
                 }
+                DiscountActivatedEvent?.Invoke(existingFlight.From, existingFlight.To);
```

Comment:
yea, now much better, requirement of someone to handle your notification forced you tu pass some data, and that is great, that is what usually happens, notification comes with data explaining how system state has changed.